[Bug]: Server Error TP on disconnect from MoveIT

### Controller Version

GF03040000

### ROS Version

jazzy

### Simulated or Real Robot?

Real

### Robot Model

A0509

### Bug Description

I am successfully able to connect to Doosan and run my custom launch script to control doosan using MoveIt2.

ros2 launch dsr_robot_driver robot.launch.py mode:=real host:=192.168.137.100

[robot.launch (1).py](https://github.com/user-attachments/files/24445433/robot.launch.1.py)

I have attached the code for my launch file as well.

On disconnect (Ctrl + c), the teach pendent is not getting the transfer back, rather  it shows a Server Error (ERROR_CONNECTION) with message "The Control server could not complete your request due to a temporary error." This message appears every time that I disconnect.

The only way to fix this is to fully restart the system and even then at times the error does not go away until for an hour or 2


I have also attached on how the error looks like 

![Image](https://github.com/user-attachments/assets/6df66e79-5224-4148-a91e-4c61d15b2d60)

https://github.com/user-attachments/assets/06b7039b-9420-46dd-a712-f58b1e6c9451

![Image](https://github.com/user-attachments/assets/278b4b41-a225-4c81-9899-4824068a8a6d)
![Image](https://github.com/user-attachments/assets/a0bb8959-e68b-4759-957f-24485676d2c7)
![Image](https://github.com/user-attachments/assets/29bf99c0-0e09-496d-926d-0a4d573723bf)

### Reproduction steps

```bash
1. Cloning the Jazzy repo

2. Building the existing packages and any custom packages

3. in my case running ros2 launch dsr_robot_driver robot.launch.py mode:=real host:=192.168.137.100 but this issue is also existing in Running ros2 launch dsr_bringup2 dsr_bringup2_rviz.launch.py mode:=real host:=192.168.137.100 port:=12345 model:=a0509

4. Shutdown (Ctrl+c)
```

### Relevant Log Output

```shell
balaji@Balaji:~/ros2_ws$ ros2 launch dsr_robot_driver robot.launch.py mode:=real host:=192.168.137.100
[INFO] [launch]: All log files can be found below /home/balaji/.ros/log/2026-01-04-14-23-57-307823-Balaji-65174
[INFO] [launch]: Default logging verbosity is set to INFO
[dsr_controller2] Loaded update_rate from YAML: 100
[dsr_controller2] Loaded update_rate from YAML: 100
[INFO] [set_config-1]: process started with pid [65177]
[INFO] [set_config-1]: process has finished cleanly [pid 65177]
[dsr_controller2] Loaded update_rate from YAML: 100
[INFO] [robot_state_publisher-2]: process started with pid [65196]
[INFO] [ros2_control_node-3]: process started with pid [65197]
[INFO] [moveit_cpp_execution_node-4]: process started with pid [65198]
[robot_state_publisher-2] [INFO] [1767516838.033797470] [dsr01.robot_state_publisher]: Robot initialized
[ros2_control_node-3] [INFO] [1767516838.040380635] [dsr01.controller_manager]: Using Steady (Monotonic) clock for triggering controller manager cycles.
[ros2_control_node-3] [INFO] [1767516838.046735298] [dsr01.controller_manager]: Subscribing to '/dsr01/robot_description' topic for robot description.
[ros2_control_node-3] [INFO] [1767516838.050214338] [dsr01.controller_manager]: update rate is 100 Hz
[ros2_control_node-3] [INFO] [1767516838.050236401] [dsr01.controller_manager]: Overruns handling is : enabled
[ros2_control_node-3] [INFO] [1767516838.050241705] [dsr01.controller_manager]: Spawning controller_manager RT thread with scheduler priority: 50
[ros2_control_node-3] [WARN] [1767516838.050303298] [dsr01.controller_manager]: Could not enable FIFO RT scheduling policy: with error number <1>(Operation not permitted). See [https://control.ros.org/master/doc/ros2_control/controller_manager/doc/userdoc.html] for details on how to enable realtime scheduling.
[ros2_control_node-3] [INFO] [1767516838.057522420] [dsr01.controller_manager]: Received robot description from topic.
[ros2_control_node-3] [INFO] [1767516838.061770201] [dsr01.controller_manager]: Loading hardware 'a0509' 
[ros2_control_node-3] [INFO] [1767516838.076899601] [dsr01.controller_manager]: Loaded hardware 'a0509' from plugin 'dsr_hardware2/DRHWInterface'
[ros2_control_node-3] [INFO] [1767516838.077080575] [dsr01.controller_manager]: Initialize hardware 'a0509' 
[ros2_control_node-3] [INFO] [1767516838.085197155] [dsr_hw_interface2]: [on_init] Mapping active joint 'joint_1' to hw_index 0
[ros2_control_node-3] [INFO] [1767516838.085227497] [dsr_hw_interface2]: [on_init] Mapping active joint 'joint_2' to hw_index 1
[ros2_control_node-3] [INFO] [1767516838.085231843] [dsr_hw_interface2]: [on_init] Mapping active joint 'joint_3' to hw_index 2
[ros2_control_node-3] [INFO] [1767516838.085234577] [dsr_hw_interface2]: [on_init] Mapping active joint 'joint_4' to hw_index 3
[ros2_control_node-3] [INFO] [1767516838.085237336] [dsr_hw_interface2]: [on_init] Mapping active joint 'joint_5' to hw_index 4
[ros2_control_node-3] [INFO] [1767516838.085239752] [dsr_hw_interface2]: [on_init] Mapping active joint 'joint_6' to hw_index 5
[ros2_control_node-3] [INFO] [1767516838.092173209] [dsr01.dsr_hw_interface2]: name: dsr01
[ros2_control_node-3] [INFO] [1767516838.092254203] [dsr01.dsr_hw_interface2]: rate: 100
[ros2_control_node-3] [INFO] [1767516838.092294630] [dsr01.dsr_hw_interface2]: standby: 5000
[ros2_control_node-3] [INFO] [1767516838.092316608] [dsr01.dsr_hw_interface2]: command: true
[ros2_control_node-3] [INFO] [1767516838.092327086] [dsr01.dsr_hw_interface2]: host: 192.168.137.100
[ros2_control_node-3] [INFO] [1767516838.092340033] [dsr01.dsr_hw_interface2]: port: 12345
[ros2_control_node-3] [INFO] [1767516838.092351141] [dsr01.dsr_hw_interface2]: mode: real
[ros2_control_node-3] [INFO] [1767516838.092389369] [dsr01.dsr_hw_interface2]: model: a0509
[ros2_control_node-3] [INFO] [1767516838.092398706] [dsr01.dsr_hw_interface2]: gripper: none
[ros2_control_node-3] [INFO] [1767516838.092432988] [dsr01.dsr_hw_interface2]: mobile: none
[ros2_control_node-3] [INFO] [1767516838.092448498] [dsr01.dsr_hw_interface2]: rt_host: 192.168.137.50
[ros2_control_node-3] [INFO] [1767516838.092489123] [dsr01.dsr_hw_interface2]: update_rate: 100
[ros2_control_node-3] [INFO] [1767516838.092567810] [dsr_hw_interface2]: _______________________________________________
[ros2_control_node-3] 
[ros2_control_node-3] [INFO] [1767516838.092581139] [dsr_hw_interface2]:     INITAILIZE
[ros2_control_node-3] [INFO] [1767516838.092583855] [dsr_hw_interface2]: _______________________________________________
[ros2_control_node-3] 
[ros2_control_node-3] [INFO] [1767516838.196888358] [dsr_hw_interface2]: Connected to DRCF
[ros2_control_node-3] [INFO] [1767516838.196990040] [dsr_hw_interface2]: INITIAL MANAGE_ACCESS_CONTROL_FORCE_REQUEST called
[ros2_control_node-3] [INFO] [1767516838.198791136] [dsr_hw_interface2]: AUTHORITY : MONITORING_ACCESS_CONTROL_GRANT
[ros2_control_node-3] [INFO] [1767516838.198805355] [dsr_hw_interface2]: INITIAL AUTHORITY GRANTED !!!
[ros2_control_node-3] [INFO] [1767516838.198811997] [dsr_hw_interface2]: ROBOT_STATE : STATE_SAFE_OFF
[ros2_control_node-3] [INFO] [1767516839.580550156] [dsr_hw_interface2]: INITIAL CONTROL_SERVO_ON called
[INFO] [spawner-5]: process started with pid [65278]
[spawner-5] [INFO] [1767516840.401208623] [dsr01.spawner_joint_state_broadcaster]: waiting for service /dsr01/controller_manager/list_controllers to become available...
[ros2_control_node-3] [INFO] [1767516840.601017986] [dsr_hw_interface2]: INITIAL CONTROL_SERVO_ON called
[ros2_control_node-3] [INFO] [1767516841.621302541] [dsr_hw_interface2]: INITIAL CONTROL_SERVO_ON called
[ros2_control_node-3] [INFO] [1767516842.641583158] [dsr_hw_interface2]: INITIAL CONTROL_SERVO_ON called
[ros2_control_node-3] [INFO] [1767516843.661919836] [dsr_hw_interface2]: INITIAL CONTROL_SERVO_ON called
[ros2_control_node-3] [INFO] [1767516844.392601289] [dsr_hw_interface2]: ROBOT_STATE : STATE_STANDBY
[ros2_control_node-3] [INFO] [1767516844.392625542] [dsr_hw_interface2]: INITIAL STATE_STANDBY !!!
[ros2_control_node-3] [INFO] [1767516844.662027224] [dsr_hw_interface2]: _______________________________________________
[ros2_control_node-3] 
[ros2_control_node-3] [INFO] [1767516844.662048908] [dsr_hw_interface2]:     OPEN CONNECTION
[ros2_control_node-3] [INFO] [1767516844.662053485] [dsr_hw_interface2]: _______________________________________________
[ros2_control_node-3] 
[ros2_control_node-3] Depreciated API.. deleted soon.
[ros2_control_node-3]  Please use 'set_on_monitoring_data_ex'
[ros2_control_node-3] Depreciated API.. deleted soon.
[ros2_control_node-3]  Please use 'set_on_monitoring_ctrl_io_ex'.
[ros2_control_node-3] [INFO] [1767516844.663320936] [dsr_hw_interface2]:     Real Robot Mode
[ros2_control_node-3] [INFO] [1767516844.663327966] [dsr_hw_interface2]:     DRCF version = GF03040000
[ros2_control_node-3] [INFO] [1767516844.663338040] [dsr_hw_interface2]:     DRFL version = GL013300
[ros2_control_node-3] [INFO] [1767516844.663341940] [dsr_hw_interface2]:     m_nVersionDRCF = 3040000
[ros2_control_node-3] [INFO] [1767516844.663346027] [dsr_hw_interface2]: _______________________________________________
[ros2_control_node-3] 
[ros2_control_node-3] usClientPort : 48658
[ros2_control_node-3] [INFO] [1767516844.806857179] [dsr_hw_interface2]: Connected RT control stream
[ros2_control_node-3] [INFO] [1767516844.807003566] [dsr_hw_interface2]: Setting velocity and acceleration limits
[ros2_control_node-3] [INFO] [1767516844.827397029] [dsr01.controller_manager]: Successful initialization of hardware 'a0509'
[ros2_control_node-3] [INFO] [1767516844.828046296] [resource_manager]: 'configure' hardware 'a0509' 
[ros2_control_node-3] [INFO] [1767516844.828057138] [resource_manager]: Successful 'configure' of hardware 'a0509'
[ros2_control_node-3] [INFO] [1767516844.828062527] [resource_manager]: 'activate' hardware 'a0509' 
[ros2_control_node-3] [INFO] [1767516844.828067432] [resource_manager]: Successful 'activate' of hardware 'a0509'
[ros2_control_node-3] [INFO] [1767516844.828124927] [dsr01.controller_manager]: Registering statistics for : a0509
[ros2_control_node-3] [INFO] [1767516844.828181770] [dsr01.controller_manager]: Resource Manager has been successfully initialized. Starting Controller Manager services...
[ros2_control_node-3] [INFO] [1767516844.906595484] [dsr01.controller_manager]: Loading controller : 'joint_state_broadcaster' of type 'joint_state_broadcaster/JointStateBroadcaster'
[ros2_control_node-3] [INFO] [1767516844.906673296] [dsr01.controller_manager]: Loading controller 'joint_state_broadcaster'
[ros2_control_node-3] [INFO] [1767516844.910842473] [dsr01.controller_manager]: Controller 'joint_state_broadcaster' node arguments: --ros-args --params-file /tmp/launch_params_8aeoo_y0 --params-file /home/balaji/ros2_master_ws/install/dsr_controller2/share/dsr_controller2/config/dsr_controller2.yaml 
[spawner-5] [INFO] [1767516844.933859449] [dsr01.spawner_joint_state_broadcaster]: Loaded joint_state_broadcaster
[ros2_control_node-3] [INFO] [1767516844.936180516] [dsr01.controller_manager]: Configuring controller: 'joint_state_broadcaster'
[ros2_control_node-3] [INFO] [1767516844.936462165] [dsr01.joint_state_broadcaster]: 'joints' or 'interfaces' parameter is empty. All available state interfaces will be published
[ros2_control_node-3] [INFO] [1767516844.954124382] [dsr01.controller_manager]: Activating controllers: [ joint_state_broadcaster ]
[ros2_control_node-3] [INFO] [1767516844.960786118] [dsr01.controller_manager]: Successfully switched controllers!
[spawner-5] [INFO] [1767516844.972517274] [dsr01.spawner_joint_state_broadcaster]: Configured and activated joint_state_broadcaster
[INFO] [spawner-5]: process has finished cleanly [pid 65278]
[INFO] [spawner-6]: process started with pid [65300]
[spawner-6] [INFO] [1767516845.585978806] [dsr01.spawner_dsr_controller2]: waiting for service /dsr01/controller_manager/list_controllers to become available...
[ros2_control_node-3] [INFO] [1767516847.088930733] [dsr01.controller_manager]: Loading controller : 'dsr_controller2' of type 'dsr_controller2/RobotController'
[ros2_control_node-3] [INFO] [1767516847.088975993] [dsr01.controller_manager]: Loading controller 'dsr_controller2'
[ros2_control_node-3] [INFO] [1767516847.109136863] [dsr_hw_interface2]: [DRFL address] 0x7e745be44ba0
[ros2_control_node-3] [INFO] [1767516847.110943379] [dsr01.controller_manager]: Controller 'dsr_controller2' node arguments: --ros-args --params-file /tmp/launch_params_8aeoo_y0 --params-file /home/balaji/ros2_master_ws/install/dsr_controller2/share/dsr_controller2/config/dsr_controller2.yaml 
[ros2_control_node-3] [INFO] [1767516847.122759793] [dsr01.dsr_controller2]: model: a0509
[spawner-6] [INFO] [1767516847.132862614] [dsr01.spawner_dsr_controller2]: Loaded dsr_controller2
[ros2_control_node-3] [INFO] [1767516847.135118007] [dsr01.controller_manager]: Configuring controller: 'dsr_controller2'
[ros2_control_node-3] [INFO] [1767516847.143703073] [dsr01.controller_manager]: Activating controllers: [ dsr_controller2 ]
[ros2_control_node-3] [INFO] [1767516847.313937095] [dsr01.controller_manager]: Successfully switched controllers!
[ros2_control_node-3] [WARN] [1767516847.313962295] [dsr01.controller_manager]: Overrun might occur, Total time : 163364.109 us (Expected < 10000.000 us) --> Read time : 28.733 us, Update time : 163312.254 us (Switch time : 163271.753 us (Switch chained mode time : 0.657 us, perform mode change time : 1.151 us, Activation time : 163266.485 us, Deactivation time : 0.839 us)), Write time : 23.122 us
[ros2_control_node-3] [WARN] [1767516847.314055304] [dsr01.controller_manager]: Overrun detected! The controller manager missed its desired rate of 100 Hz. The loop took 163.535941 ms (missed cycles : 17).
[ros2_control_node-3] [WARN] [1767516847.320662719] [dsr_hw_interface2]: [REAL] Skip dt=0.169993 (expected=0.010000, allowed=[0.003000, 0.015000])
[spawner-6] [INFO] [1767516847.322913827] [dsr01.spawner_dsr_controller2]: Configured and activated dsr_controller2
[INFO] [spawner-6]: process has finished cleanly [pid 65300]
[INFO] [spawner-7]: process started with pid [65315]
[spawner-7] [INFO] [1767516847.928395386] [dsr01.spawner_dsr_moveit_controller]: waiting for service /dsr01/controller_manager/list_controllers to become available...
[ros2_control_node-3] [INFO] [1767516849.483388447] [dsr01.controller_manager]: Loading controller : 'dsr_moveit_controller' of type 'joint_trajectory_controller/JointTrajectoryController'
[ros2_control_node-3] [INFO] [1767516849.483432732] [dsr01.controller_manager]: Loading controller 'dsr_moveit_controller'
[ros2_control_node-3] [INFO] [1767516849.486350347] [dsr01.controller_manager]: Controller 'dsr_moveit_controller' node arguments: --ros-args --params-file /tmp/launch_params_8aeoo_y0 --params-file /home/balaji/ros2_master_ws/install/dsr_controller2/share/dsr_controller2/config/dsr_controller2.yaml 
[spawner-7] [INFO] [1767516849.521987457] [dsr01.spawner_dsr_moveit_controller]: Loaded dsr_moveit_controller
[ros2_control_node-3] [INFO] [1767516849.523514815] [dsr01.controller_manager]: Configuring controller: 'dsr_moveit_controller'
[ros2_control_node-3] [INFO] [1767516849.523823911] [dsr01.dsr_moveit_controller]: No specific joint names are used for command interfaces. Using 'joints' parameter.
[ros2_control_node-3] [INFO] [1767516849.523862008] [dsr01.dsr_moveit_controller]: Command interfaces are [position velocity] and state interfaces are [position velocity].
[ros2_control_node-3] [INFO] [1767516849.523896503] [dsr01.dsr_moveit_controller]: Using 'splines' interpolation method.
[ros2_control_node-3] [INFO] [1767516849.528312135] [dsr01.dsr_moveit_controller]: Action status changes will be monitored at 20.00 Hz.
[ros2_control_node-3] [INFO] [1767516849.543413902] [dsr01.controller_manager]: Activating controllers: [ dsr_moveit_controller ]
[ros2_control_node-3] [INFO] [1767516849.551037765] [dsr01.controller_manager]: Successfully switched controllers!
[spawner-7] [INFO] [1767516849.562343837] [dsr01.spawner_dsr_moveit_controller]: Configured and activated dsr_moveit_controller
[INFO] [spawner-7]: process has finished cleanly [pid 65315]
[INFO] [move_group-8]: process started with pid [65331]
[INFO] [rviz2-9]: process started with pid [65332]
[move_group-8] [INFO] [1767516850.058649198] [dsr01.moveit.moveit.ros.rdf_loader]: Loaded robot model in 0.00349717 seconds
[move_group-8] [INFO] [1767516850.058796025] [dsr01.moveit.moveit.core.robot_model]: Loading robot model 'scanner_env'...
[move_group-8] [WARN] [1767516850.281054672] [dsr01.moveit.moveit.core.robot_model]: Link spot_light_link has visual geometry but no collision geometry. Collision geometry will be left empty. Fix your URDF file by explicitly specifying collision geometry.
[rviz2-9] [INFO] [1767516850.333384931] [rviz2]: Stereo is NOT SUPPORTED
[rviz2-9] [INFO] [1767516850.333451890] [rviz2]: OpenGl version: 4.6 (GLSL 4.6)
[rviz2-9] [INFO] [1767516850.354425824] [rviz2]: Stereo is NOT SUPPORTED
[move_group-8] [INFO] [1767516850.485136879] [trac_ik_kinematics_plugin.trac_ik_kinematics_plugin]: IK Using joint link_1 -6.2832 6.2832
[move_group-8] [INFO] [1767516850.485165312] [trac_ik_kinematics_plugin.trac_ik_kinematics_plugin]: IK Using joint link_2 -6.2832 6.2832
[move_group-8] [INFO] [1767516850.485172000] [trac_ik_kinematics_plugin.trac_ik_kinematics_plugin]: IK Using joint link_3 -2.7925 2.7925
[move_group-8] [INFO] [1767516850.485176991] [trac_ik_kinematics_plugin.trac_ik_kinematics_plugin]: IK Using joint link_4 -6.2832 6.2832
[move_group-8] [INFO] [1767516850.485182076] [trac_ik_kinematics_plugin.trac_ik_kinematics_plugin]: IK Using joint link_5 -6.2832 6.2832
[move_group-8] [INFO] [1767516850.485186485] [trac_ik_kinematics_plugin.trac_ik_kinematics_plugin]: IK Using joint link_6 -6.2832 6.2832
[move_group-8] [INFO] [1767516850.485190354] [trac_ik_kinematics_plugin.trac_ik_kinematics_plugin]: Using solve type Speed
[rviz2-9] Warning: class_loader.impl: SEVERE WARNING!!! A namespace collision has occurred with plugin factory for class rviz_default_plugins::displays::InteractiveMarkerDisplay. New factory will OVERWRITE existing one. This situation occurs when libraries containing plugins are directly linked against an executable (the one running right now generating this message). Please separate plugins out into their own library or just don't link against the library and use either class_loader::ClassLoader/MultiLibraryClassLoader to open.
[rviz2-9]          at line 321 in /opt/ros/jazzy/include/class_loader/class_loader/class_loader_core.hpp
[rviz2-9] [INFO] [1767516850.630542589] [dsr01.rviz]: Subscribing to: /nbv_dbscan/low_density_cloud
[rviz2-9] 
[rviz2-9] [INFO] [1767516850.644809811] [dsr01.rviz]: Subscribing to: /nbv_dbscan/high_density_cloud
[rviz2-9] 
[move_group-8] [INFO] [1767516851.215551601] [dsr01.moveit.moveit.ros.planning_scene_monitor]: Publishing maintained planning scene on 'monitored_planning_scene'
[move_group-8] [INFO] [1767516851.215734397] [dsr01.moveit.moveit.ros.moveit_cpp]: Listening to 'joint_states' for joint states
[move_group-8] [INFO] [1767516851.217149993] [dsr01.moveit.moveit.ros.current_state_monitor]: Listening to joint states on topic 'joint_states'
[move_group-8] [INFO] [1767516851.217710005] [dsr01.moveit.moveit.ros.planning_scene_monitor]: Listening to '/dsr01/attached_collision_object' for attached collision objects
[move_group-8] [INFO] [1767516851.217733753] [dsr01.moveit.moveit.ros.planning_scene_monitor]: Stopping existing planning scene publisher.
[move_group-8] [INFO] [1767516851.217841997] [dsr01.moveit.moveit.ros.planning_scene_monitor]: Stopped publishing maintained planning scene.
[move_group-8] [INFO] [1767516851.218275548] [dsr01.moveit.moveit.ros.planning_scene_monitor]: Publishing maintained planning scene on 'monitored_planning_scene'
[move_group-8] [INFO] [1767516851.218377381] [dsr01.moveit.moveit.ros.planning_scene_monitor]: Starting planning scene monitor
[move_group-8] [INFO] [1767516851.218851274] [dsr01.moveit.moveit.ros.planning_scene_monitor]: Listening to '/dsr01/planning_scene'
[move_group-8] [INFO] [1767516851.218876003] [dsr01.moveit.moveit.ros.planning_scene_monitor]: Starting world geometry update monitor for collision objects, attached objects, octomap updates.
[move_group-8] [INFO] [1767516851.220306242] [dsr01.moveit.moveit.ros.planning_scene_monitor]: Listening to 'collision_object'
[move_group-8] [INFO] [1767516851.220818235] [dsr01.moveit.moveit.ros.planning_scene_monitor]: Listening to 'planning_scene_world' for planning scene world geometry
[move_group-8] [WARN] [1767516851.221221702] [dsr01.moveit.moveit.ros.occupancy_map_monitor]: Resolution not specified for Octomap. Assuming resolution = 0.1 instead
[move_group-8] [ERROR] [1767516851.221248081] [dsr01.moveit.moveit.ros.occupancy_map_monitor]: No 3D sensor plugin(s) defined for octomap updates
[move_group-8] [INFO] [1767516851.389200348] [dsr01.moveit.moveit.ros.planning_pipeline]: Successfully loaded planner 'OMPL'
[move_group-8] [INFO] [1767516851.391540201] [dsr01.move_group]: Try loading adapter 'default_planning_request_adapters/ResolveConstraintFrames'
[move_group-8] [INFO] [1767516851.393326526] [dsr01.move_group]: Loaded adapter 'default_planning_request_adapters/ResolveConstraintFrames'
[move_group-8] [INFO] [1767516851.393351124] [dsr01.move_group]: Try loading adapter 'default_planning_request_adapters/ValidateWorkspaceBounds'
[move_group-8] [INFO] [1767516851.393611098] [dsr01.move_group]: Loaded adapter 'default_planning_request_adapters/ValidateWorkspaceBounds'
[move_group-8] [INFO] [1767516851.393624841] [dsr01.move_group]: Try loading adapter 'default_planning_request_adapters/CheckStartStateBounds'
[move_group-8] [INFO] [1767516851.393663537] [dsr01.move_group]: Loaded adapter 'default_planning_request_adapters/CheckStartStateBounds'
[move_group-8] [INFO] [1767516851.393672196] [dsr01.move_group]: Try loading adapter 'default_planning_request_adapters/CheckStartStateCollision'
[move_group-8] [INFO] [1767516851.393691034] [dsr01.move_group]: Loaded adapter 'default_planning_request_adapters/CheckStartStateCollision'
[move_group-8] [INFO] [1767516851.395322064] [dsr01.move_group]: Try loading adapter 'default_planning_response_adapters/AddTimeOptimalParameterization'
[move_group-8] [INFO] [1767516851.398219919] [dsr01.move_group]: Loaded adapter 'default_planning_response_adapters/AddTimeOptimalParameterization'
[move_group-8] [INFO] [1767516851.398284182] [dsr01.move_group]: Try loading adapter 'default_planning_response_adapters/ValidateSolution'
[move_group-8] [INFO] [1767516851.402675005] [dsr01.move_group]: Loaded adapter 'default_planning_response_adapters/ValidateSolution'
[move_group-8] [INFO] [1767516851.402755153] [dsr01.move_group]: Try loading adapter 'default_planning_response_adapters/DisplayMotionPath'
[move_group-8] [INFO] [1767516851.404248205] [dsr01.move_group]: Loaded adapter 'default_planning_response_adapters/DisplayMotionPath'
[move_group-8] [INFO] [1767516851.412064231] [dsr01.moveit.moveit.ros.planning_pipeline]: Successfully loaded planner 'CHOMP'
[move_group-8] [INFO] [1767516851.414637256] [dsr01.move_group]: Try loading adapter 'default_planning_request_adapters/ResolveConstraintFrames'
[move_group-8] [INFO] [1767516851.415117917] [dsr01.move_group]: Loaded adapter 'default_planning_request_adapters/ResolveConstraintFrames'
[move_group-8] [INFO] [1767516851.415163673] [dsr01.move_group]: Try loading adapter 'default_planning_request_adapters/ValidateWorkspaceBounds'
[move_group-8] [INFO] [1767516851.415844265] [dsr01.move_group]: Loaded adapter 'default_planning_request_adapters/ValidateWorkspaceBounds'
[move_group-8] [INFO] [1767516851.415891692] [dsr01.move_group]: Try loading adapter 'default_planning_request_adapters/CheckStartStateBounds'
[move_group-8] [INFO] [1767516851.415975375] [dsr01.move_group]: Loaded adapter 'default_planning_request_adapters/CheckStartStateBounds'
[move_group-8] [INFO] [1767516851.415983330] [dsr01.move_group]: Try loading adapter 'default_planning_request_adapters/CheckStartStateCollision'
[move_group-8] [INFO] [1767516851.416009429] [dsr01.move_group]: Loaded adapter 'default_planning_request_adapters/CheckStartStateCollision'
[move_group-8] [INFO] [1767516851.417918854] [dsr01.move_group]: Try loading adapter 'default_planning_response_adapters/AddTimeOptimalParameterization'
[move_group-8] [INFO] [1767516851.418614644] [dsr01.move_group]: Loaded adapter 'default_planning_response_adapters/AddTimeOptimalParameterization'
[move_group-8] [INFO] [1767516851.418644699] [dsr01.move_group]: Try loading adapter 'default_planning_response_adapters/ValidateSolution'
[move_group-8] [INFO] [1767516851.419700653] [dsr01.move_group]: Loaded adapter 'default_planning_response_adapters/ValidateSolution'
[move_group-8] [INFO] [1767516851.419726896] [dsr01.move_group]: Try loading adapter 'default_planning_response_adapters/DisplayMotionPath'
[move_group-8] [INFO] [1767516851.420342615] [dsr01.move_group]: Loaded adapter 'default_planning_response_adapters/DisplayMotionPath'
[move_group-8] [INFO] [1767516851.424419372] [dsr01.moveit.moveit.planners.pilz.joint_limits_aggregator]: Reading limits from namespace robot_description_planning
[move_group-8] [INFO] [1767516851.429762945] [dsr01.moveit.moveit.planners.pilz.command_planner]: Available plugins: pilz_industrial_motion_planner/PlanningContextLoaderCIRC pilz_industrial_motion_planner/PlanningContextLoaderLIN pilz_industrial_motion_planner/PlanningContextLoaderPTP 
[move_group-8] [INFO] [1767516851.429831778] [dsr01.moveit.moveit.planners.pilz.command_planner]: About to load: pilz_industrial_motion_planner/PlanningContextLoaderCIRC
[move_group-8] [INFO] [1767516851.432357642] [dsr01.moveit.moveit.planners.pilz.command_planner]: Registered Algorithm [CIRC]
[move_group-8] [INFO] [1767516851.432429168] [dsr01.moveit.moveit.planners.pilz.command_planner]: About to load: pilz_industrial_motion_planner/PlanningContextLoaderLIN
[move_group-8] [INFO] [1767516851.433934151] [dsr01.moveit.moveit.planners.pilz.command_planner]: Registered Algorithm [LIN]
[move_group-8] [INFO] [1767516851.433987640] [dsr01.moveit.moveit.planners.pilz.command_planner]: About to load: pilz_industrial_motion_planner/PlanningContextLoaderPTP
[move_group-8] [INFO] [1767516851.435223018] [dsr01.moveit.moveit.planners.pilz.command_planner]: Registered Algorithm [PTP]
[move_group-8] [INFO] [1767516851.435263061] [dsr01.moveit.moveit.ros.planning_pipeline]: Successfully loaded planner 'Pilz Industrial Motion Planner'
[move_group-8] [INFO] [1767516851.436942400] [dsr01.move_group]: Try loading adapter 'default_planning_request_adapters/ResolveConstraintFrames'
[move_group-8] [INFO] [1767516851.437094386] [dsr01.move_group]: Loaded adapter 'default_planning_request_adapters/ResolveConstraintFrames'
[move_group-8] [INFO] [1767516851.437100643] [dsr01.move_group]: Try loading adapter 'default_planning_request_adapters/ValidateWorkspaceBounds'
[move_group-8] [INFO] [1767516851.437401747] [dsr01.move_group]: Loaded adapter 'default_planning_request_adapters/ValidateWorkspaceBounds'
[move_group-8] [INFO] [1767516851.437422520] [dsr01.move_group]: Try loading adapter 'default_planning_request_adapters/CheckStartStateBounds'
[move_group-8] [INFO] [1767516851.437456650] [dsr01.move_group]: Loaded adapter 'default_planning_request_adapters/CheckStartStateBounds'
[move_group-8] [INFO] [1767516851.437463880] [dsr01.move_group]: Try loading adapter 'default_planning_request_adapters/CheckStartStateCollision'
[move_group-8] [INFO] [1767516851.437482898] [dsr01.move_group]: Loaded adapter 'default_planning_request_adapters/CheckStartStateCollision'
[move_group-8] [INFO] [1767516851.439084565] [dsr01.move_group]: Try loading adapter 'default_planning_response_adapters/AddTimeOptimalParameterization'
[move_group-8] [INFO] [1767516851.439918653] [dsr01.move_group]: Loaded adapter 'default_planning_response_adapters/AddTimeOptimalParameterization'
[move_group-8] [INFO] [1767516851.439939137] [dsr01.move_group]: Try loading adapter 'default_planning_response_adapters/ValidateSolution'
[move_group-8] [INFO] [1767516851.441077355] [dsr01.move_group]: Loaded adapter 'default_planning_response_adapters/ValidateSolution'
[move_group-8] [INFO] [1767516851.441104144] [dsr01.move_group]: Try loading adapter 'default_planning_response_adapters/DisplayMotionPath'
[move_group-8] [INFO] [1767516851.442245747] [dsr01.move_group]: Loaded adapter 'default_planning_response_adapters/DisplayMotionPath'
[move_group-8] [INFO] [1767516851.509996593] [dsr01.moveit.moveit.plugins.simple_controller_manager]: Added FollowJointTrajectory controller for dsr_moveit_controller
[move_group-8] [INFO] [1767516851.510217644] [dsr01.moveit.moveit.plugins.simple_controller_manager]: Returned 1 controllers in list
[move_group-8] [INFO] [1767516851.510248003] [dsr01.moveit.moveit.plugins.simple_controller_manager]: Returned 1 controllers in list
[move_group-8] [INFO] [1767516851.511098795] [dsr01.moveit.moveit.ros.trajectory_execution_manager]: Trajectory execution is managing controllers
[move_group-8] [INFO] [1767516851.511131518] [dsr01.move_group]: MoveGroup debug mode is ON
[move_group-8] [INFO] [1767516851.543846571] [dsr01.moveit.moveit.planners.pilz.move_group_sequence_action]: initialize move group sequence action
[move_group-8] [INFO] [1767516851.548295192] [dsr01.moveit.moveit.planners.pilz.joint_limits_aggregator]: Reading limits from namespace robot_description_planning
[move_group-8] [INFO] [1767516851.548622893] [dsr01.moveit.moveit.planners.pilz.joint_limits_aggregator]: Reading limits from namespace robot_description_planning
[move_group-8] [INFO] [1767516851.549768401] [dsr01.moveit.moveit.ros.move_group.executable]: 
[move_group-8] 
[move_group-8] ********************************************************
[move_group-8] * MoveGroup using: 
[move_group-8] *     - apply_planning_scene_service
[move_group-8] *     - clear_octomap_service
[move_group-8] *     - get_group_urdf
[move_group-8] *     - load_geometry_from_file
[move_group-8] *     - CartesianPathService
[move_group-8] *     - execute_trajectory_action
[move_group-8] *     - get_planning_scene_service
[move_group-8] *     - kinematics_service
[move_group-8] *     - move_action
[move_group-8] *     - motion_plan_service
[move_group-8] *     - query_planners_service
[move_group-8] *     - state_validation_service
[move_group-8] *     - save_geometry_to_file
[move_group-8] *     - SequenceAction
[move_group-8] *     - SequenceService
[move_group-8] ********************************************************
[move_group-8] 
[move_group-8] [INFO] [1767516851.549873405] [dsr01.moveit.moveit.ros.move_group.context]: MoveGroup context using pipeline ompl
[move_group-8] [INFO] [1767516851.549884681] [dsr01.moveit.moveit.ros.move_group.context]: MoveGroup context initialization complete
[move_group-8] Loading 'move_group/ApplyPlanningSceneService'...
[move_group-8] Loading 'move_group/ClearOctomapService'...
[move_group-8] Loading 'move_group/GetUrdfService'...
[move_group-8] Loading 'move_group/LoadGeometryFromFileService'...
[move_group-8] Loading 'move_group/MoveGroupCartesianPathService'...
[move_group-8] Loading 'move_group/MoveGroupExecuteTrajectoryAction'...
[move_group-8] Loading 'move_group/MoveGroupGetPlanningSceneService'...
[move_group-8] Loading 'move_group/MoveGroupKinematicsService'...
[move_group-8] Loading 'move_group/MoveGroupMoveAction'...
[move_group-8] Loading 'move_group/MoveGroupPlanService'...
[move_group-8] Loading 'move_group/MoveGroupQueryPlannersService'...
[move_group-8] Loading 'move_group/MoveGroupStateValidationService'...
[move_group-8] Loading 'move_group/SaveGeometryToFileService'...
[move_group-8] Loading 'pilz_industrial_motion_planner/MoveGroupSequenceAction'...
[move_group-8] Loading 'pilz_industrial_motion_planner/MoveGroupSequenceService'...
[move_group-8] 
[move_group-8] You can start planning now!
[move_group-8] 
[rviz2-9] [ERROR] [1767516853.685853316] [dsr01.moveit_331089702.moveit.ros.motion_planning_frame]: Action server: /recognize_objects not available
[rviz2-9] [INFO] [1767516853.711249890] [dsr01.moveit_331089702.moveit.ros.motion_planning_frame]: MoveGroup namespace changed: /dsr01 -> . Reloading params.
[rviz2-9] [INFO] [1767516853.739469888] [dsr01.rviz]: Subscribing to: /nbv_dbscan/low_density_cloud
[rviz2-9] 
[rviz2-9] [INFO] [1767516853.744763111] [dsr01.rviz]: Subscribing to: /nbv_dbscan/high_density_cloud
[rviz2-9] 
[rviz2-9] [INFO] [1767516853.858290524] [rviz2]: Stereo is NOT SUPPORTED
[rviz2-9] [INFO] [1767516853.882225233] [dsr01.moveit_331089702.moveit.ros.rdf_loader]: Loaded robot model in 0.00722971 seconds
[rviz2-9] [INFO] [1767516853.882453401] [dsr01.moveit_331089702.moveit.core.robot_model]: Loading robot model 'scanner_env'...
[rviz2-9] [WARN] [1767516854.158499944] [dsr01.moveit_331089702.moveit.core.robot_model]: Link spot_light_link has visual geometry but no collision geometry. Collision geometry will be left empty. Fix your URDF file by explicitly specifying collision geometry.
[rviz2-9] [INFO] [1767516854.166946676] [trac_ik_kinematics_plugin.trac_ik_kinematics_plugin]: IK Using joint link_1 -6.2832 6.2832
[rviz2-9] [INFO] [1767516854.166977355] [trac_ik_kinematics_plugin.trac_ik_kinematics_plugin]: IK Using joint link_2 -6.2832 6.2832
[rviz2-9] [INFO] [1767516854.166983226] [trac_ik_kinematics_plugin.trac_ik_kinematics_plugin]: IK Using joint link_3 -2.7925 2.7925
[rviz2-9] [INFO] [1767516854.166987777] [trac_ik_kinematics_plugin.trac_ik_kinematics_plugin]: IK Using joint link_4 -6.2832 6.2832
[rviz2-9] [INFO] [1767516854.166992065] [trac_ik_kinematics_plugin.trac_ik_kinematics_plugin]: IK Using joint link_5 -6.2832 6.2832
[rviz2-9] [INFO] [1767516854.166996416] [trac_ik_kinematics_plugin.trac_ik_kinematics_plugin]: IK Using joint link_6 -6.2832 6.2832
[rviz2-9] [INFO] [1767516854.167001071] [trac_ik_kinematics_plugin.trac_ik_kinematics_plugin]: Using solve type Speed
[rviz2-9] [INFO] [1767516854.876030267] [dsr01.moveit_331089702.moveit.ros.planning_scene_monitor]: Publishing maintained planning scene on 'monitored_planning_scene'
[rviz2-9] [INFO] [1767516856.044008567] [dsr01.moveit_331089702.moveit.ros.planning_scene_monitor]: Starting planning scene monitor
[rviz2-9] [INFO] [1767516856.045065864] [dsr01.moveit_331089702.moveit.ros.planning_scene_monitor]: Listening to '/dsr01/monitored_planning_scene'
[rviz2-9] [INFO] [1767516856.058241221] [dsr01.interactive_marker_display_99528463283904]: Connected on namespace: /dsr01/rviz_moveit_motion_planning_display/robot_interaction_interactive_marker_topic
[rviz2-9] [INFO] [1767516856.136597352] [dsr01.interactive_marker_display_99528463283904]: Sending request for interactive markers
[rviz2-9] [INFO] [1767516856.143187188] [dsr01.moveit_331089702.moveit.ros.planning_scene_monitor]: Stopped publishing maintained planning scene.
[rviz2-9] [INFO] [1767516856.143310027] [dsr01.moveit_331089702.moveit.ros.planning_scene_monitor]: Stopping planning scene monitor
[rviz2-9] [INFO] [1767516856.174008037] [dsr01.moveit_331089702.moveit.ros.rdf_loader]: Loaded robot model in 0.00593606 seconds
[rviz2-9] [INFO] [1767516856.174102261] [dsr01.moveit_331089702.moveit.core.robot_model]: Loading robot model 'scanner_env'...
[rviz2-9] [WARN] [1767516856.214818455] [dsr01.interactive_marker_display_99528463283904]: Server not available during initialization, resetting
[rviz2-9] [WARN] [1767516856.460068547] [dsr01.moveit_331089702.moveit.core.robot_model]: Link spot_light_link has visual geometry but no collision geometry. Collision geometry will be left empty. Fix your URDF file by explicitly specifying collision geometry.
[rviz2-9] [INFO] [1767516856.463524896] [trac_ik_kinematics_plugin.trac_ik_kinematics_plugin]: IK Using joint link_1 -6.2832 6.2832
[rviz2-9] [INFO] [1767516856.463552219] [trac_ik_kinematics_plugin.trac_ik_kinematics_plugin]: IK Using joint link_2 -6.2832 6.2832
[rviz2-9] [INFO] [1767516856.463556131] [trac_ik_kinematics_plugin.trac_ik_kinematics_plugin]: IK Using joint link_3 -2.7925 2.7925
[rviz2-9] [INFO] [1767516856.463559508] [trac_ik_kinematics_plugin.trac_ik_kinematics_plugin]: IK Using joint link_4 -6.2832 6.2832
[rviz2-9] [INFO] [1767516856.463562743] [trac_ik_kinematics_plugin.trac_ik_kinematics_plugin]: IK Using joint link_5 -6.2832 6.2832
[rviz2-9] [INFO] [1767516856.463565820] [trac_ik_kinematics_plugin.trac_ik_kinematics_plugin]: IK Using joint link_6 -6.2832 6.2832
[rviz2-9] [INFO] [1767516856.463569104] [trac_ik_kinematics_plugin.trac_ik_kinematics_plugin]: Using solve type Speed
[rviz2-9] [INFO] [1767516857.209476249] [dsr01.moveit_331089702.moveit.ros.planning_scene_monitor]: Publishing maintained planning scene on 'monitored_planning_scene'
[rviz2-9] [INFO] [1767516857.240927779] [dsr01.moveit_331089702.moveit.ros.planning_scene_monitor]: Starting planning scene monitor
[rviz2-9] [INFO] [1767516857.242883381] [dsr01.moveit_331089702.moveit.ros.planning_scene_monitor]: Listening to '/dsr01/monitored_planning_scene'
[rviz2-9] [INFO] [1767516857.279995878] [dsr01.moveit_331089702.moveit.ros.motion_planning_frame]: group arm
[rviz2-9] [INFO] [1767516857.280072605] [dsr01.moveit_331089702.moveit.ros.motion_planning_frame]: Constructing new MoveGroup connection for group 'arm' in namespace '/dsr01'
[rviz2-9] [INFO] [1767516857.284869502] [dsr01.interactive_marker_display_99528463283904]: Sending request for interactive markers
[rviz2-9] [INFO] [1767516857.307343550] [dsr01.moveit_331089702.moveit.ros.move_group_interface]: Ready to take commands for planning group arm.
[rviz2-9] [INFO] [1767516857.315343871] [dsr01.moveit_331089702.moveit.ros.motion_planning_frame]: group arm
[rviz2-9] [INFO] [1767516857.355496419] [dsr01.interactive_marker_display_99528463283904]: Service response received for initialization
[move_group-8] [INFO] [1767516870.774562039] [dsr01.moveit.moveit.ros.move_group.move_action]: MoveGroupMoveAction: Received request
[rviz2-9] [INFO] [1767516870.774854667] [dsr01.moveit_331089702.moveit.ros.move_group_interface]: Plan and Execute request accepted
[move_group-8] [INFO] [1767516870.775020969] [dsr01.moveit.moveit.ros.move_group.move_action]: executing..
```

--- a/dsr_controller2/src/dsr_controller2.cpp
+++ b/dsr_controller2/src/dsr_controller2.cpp
@@ -1921,7 +1921,8 @@ auto connect_rt_control_cb = [this](const std::shared_ptr<dsr_msgs2::srv::Connec
 auto disconnect_rt_control_cb = [this](const std::shared_ptr<dsr_msgs2::srv::DisconnectRtControl::Request> /*req*/, 
                                        std::shared_ptr<dsr_msgs2::srv::DisconnectRtControl::Response> res) -> void 
 {
-    res->success = Drfl->disconnect_rt_control();
+    // To-do : Update disconnection function in controller version v3.6
+    // res->success = Drfl->disconnect_rt_control();
 };
 
 auto get_rt_control_output_version_list_cb = [this](const std::shared_ptr<dsr_msgs2::srv::GetRtControlOutputVersionList::Request> /*req*/, 

--- a/dsr_hardware2/src/dsr_hw_interface2.cpp
+++ b/dsr_hardware2/src/dsr_hw_interface2.cpp
@@ -720,9 +720,10 @@ return_type DRHWInterface::write(const rclcpp::Time &, const rclcpp::Duration &d
 
 DRHWInterface::~DRHWInterface()
 {
-    Drfl.stop_rt_control();
-    Drfl.disconnect_rt_control();
-    Drfl.close_connection();
+	Drfl.stop_rt_control();
+	// To-do : Update disconnection function in controller version v3.6
+	// Drfl.disconnect_rt_control();
+	Drfl.close_connection();
 
     RCLCPP_INFO(rclcpp::get_logger("dsr_hw_interface2"),"_______________________________________________\n"); 
     RCLCPP_INFO(rclcpp::get_logger("dsr_hw_interface2"),"    CONNECTION IS CLOSED");

